frontend/src/Components/Admin/Admin-form.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { history } from "react-router-dom";
+export default class Form extends React.Component {
+    state = {
+        lesson: "",
+        event_date: "",
+        description: ""
+    };
+    constructor(props) {
+        super(props);
+        this.lessonRef = React.createRef();
+        this.event_dateRef = React.createRef();
+        this.descriptionRef = React.createRef();
+    }
+
+    onSubmit = event => {
+        event.preventDefault();
+        const body = {
+            lesson: this.lessonRef.current.value,
+            event_date: this.event_dateRef.current.value,
+            description: this.descriptionRef.current.value
+        };
+        fetch("/api/v1/events", {
```

Comment:
@Behnazz put the `this.props.history.push("/events")` in a then, else the page might navigate to the events page before adding the event.
e.g
```
fetch('http://example.com/movies.json')
  .then(function(myJson) {
    console.log(JSON.stringify(myJson));
  });